tfx_addons/pandas_transform/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Init module for PandasTransform"""
+
+from tfx_addons.pandas_transform.component import PandasTransform
+
+__version__ = '0.1dev'
```

Comment:
@rcrowe-google Doesn't the number should match the version in `Release.md`?

Reply:
Doh! Yes, it should.  Fixed.  Thanks!